src/infrastructure/validation/testing/TestCase.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * @author - Abhishek Saran(111701001)
+ * @module - Infrastucture module 
+ * @team - Test Harness
+ * @description - Abstract class which implements ITest Interface
+ * @summary - Modules have to extend the abstract class to implement the run method for each test case  
+ */
+
+package infrastructure.validation.testing;
+
+public abstract class TestCase implements ITest {
+
+	//data members
+	/*
+	 * @category - of a test case as its module name 
+	 * @error - occured in the test case 
+	 * @description - a small description about what this test case is about
+	 * @priority - the priority level of the test case - 0 being lowest and 2 being highest
+	 */
```

Comment:
What starts with @ is a block tag. Only @param is a valid block tag. Use one param block tag, put the name of the data member and write the description in the same line and put each param in a new line using 4 spaces to indent it from the @.

Reply:
> Please add a package-info.java file in the testing folder like done [here](https://github.com/abhisheksaran/KoraKaagaz/blob/51-ITest_interface/src/infrastructure/validation/logger/package-info.java). The folder name "testing" is our package and the files in this folder is what gets imported when someone calls import infrastructure.validation.testing. We are putting our package files here and writing in that info file. In @author, you can put your name.

What should be written in this info file?